### ISSUE TITLE
Right to left language support

### ISSUE THREAD

**Issue by benwiley4000** [benwiley4000/react-responsive-audio-player#247]
We should have a boolean which can be passed to the default player to make text right-to-left instead of left-to-right.

In order to support right-to-left languages we need to basically flip the whole UI for the default player horizontally. Changes required will include:

- [x] `MaybeMarquee` component should be able to scroll right instead of left.
- [ ] The volume and shuffle icons should be flipped horizontally in right-to-left mode. The repeat button and the media playback icons should stay the same and in the same order.. 
- [ ] When in right-to-left mode, the order of controls should be reversed, and flex-wrap should become `wrap` instead of `wrap-reverse`. This way we can horizontally flip the order of controls without affecting the row ordering.
  - [ ] Important exception is that play/pause/skip controls (including the media progress bar) should maintain their direction and order. So maybe we should add a new keyword for the `controls` array which encompasses all three and renders them as a React Fragment so they stay intact for the RTL layout.
- [ ] The placement order of the marquee and the time ratio in the audio status bar should be reversed in right-to-left mode
- [ ] The `timeElapsed / totalTime` format might not localize well to other languages, especially RTL.. instead of trying to solve that one ourselves we should make it possible to custom-specify the format.
  - [ ] Needed: A name for what that time progress thing is actually called
- [ ] Have some people fluent in RTL languages review this to make sure it makes sense (might want to spin off into separate issue once the initial work is done)

And if we ever add support for a horizontal volume bar (maybe we shouldn't), then it would need to run in reverse in right-to-left mode.

**Comment by benwiley4000** [comment]
I want to work on this but it's not reasonable for the 2.0 release as it's not going to be in high demand (yet, anyway).
